bench/index.ts

```diff
@@ -94,18 +119,21 @@ void bench({
     string key
     string value
     `,
-  msg: {
-    header: {
-      seq: 0,
-      stamp: { sec: 0, nsec: 0 },
-      frame_id: "frame id",
+    msg: {
+      header: {
+        seq: 0,
+        stamp: { sec: 0, nsec: 0 },
+        frame_id: "frame id",
+      },
+      status: new Array(20).fill({
+        level: 0,
+        name: "some name",
+        message: "some message usually longer",
+        hardware_id: "some hardware id",
+        values: new Array(20).fill({ key: "a key", value: "some value" }),
+      }),
     },
-    status: new Array(20).fill({
-      level: 0,
-      name: "some name",
-      message: "some message usually longer",
-      hardware_id: "some hardware id",
-      values: new Array(20).fill({ key: "a key", value: "some value" }),
-    }),
-  },
-});
+  });
+}
+
+void main();
```

Comment:
a better approach here would be to `main().catch()` like we do here:

https://github.com/foxglove/studio/blob/main/ci/storybook.ts#L165-L170

Reply:
Without a catch the error is re-thrown into the process space as an uncaught exception which is what I'm ok with happening here.